datadog-cloudformation-common/pom.xml

```diff
@@ -20,6 +27,6 @@
         </dependency>
     </dependencies>
     <properties>
-        <datadog-api-client-version>1.0.0</datadog-api-client-version>
+        <datadog-api-client-version>1.0.0-beta.4-SNAPSHOT</datadog-api-client-version>
```

Comment:
```suggestion
        <datadog-api-client-version>1.0.0-beta.3-SNAPSHOT</datadog-api-client-version>
```

Reply:
Actually this shouldn't even be a snapshot, we should just use `1.0.0-beta.3`.